api/swagger/swagger.yaml

```diff
@@ -147,31 +147,50 @@ paths:
           description: Error
           schema:
             $ref: "#/definitions/ErrorResponse"
-  /event/{page},{pageSize}:
-      x-swagger-router-controller: EventController
-      get:
-        description: get a page of events
-        operationId: getEvents
-        parameters:
-          - name: page
-            type: number
-            in: path
-            required: true
-          - name: pageSize
-            type: number
-            in: path
-            required: true
-            default: 10
-        responses:
-          "200":
-            description: Success
-            schema:
-              $ref: "#/definitions/EventListResponse"
-          # responses may fall through to errors
-          default:
-            description: Error
-            schema:
-              $ref: "#/definitions/ErrorResponse"
+  /events/:
+    x-swagger-router-controller: EventController
+    get:
+      description: get pagenated events within a radius or by created time if user location is not provided
+      operationId: getEvents
+      parameters:
+        - name: sortBy
+          type: string
+          in: header
+          default: ""
```

Comment:
can it be optional?

Reply:
I don't think so.
https://stackoverflow.com/questions/35011192/how-to-define-an-optional-parameter-in-path-using-swagger

"Given that path parameter must be required according to the OpenAPI/Swagger spec, you can consider adding 2 separate endpoints with the following paths:"